readthedocs.yml

```diff
@@ -0,0 +1,17 @@
+# readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+sphinx:
+  builder: html
+  configuration: docs/conf.py
+  fail_on_warning: false
+
+python:
+   install:
+      - method: pip
+        extra_requirements:
+           - dev
+        path: .
```

Comment:
Need to wait for this to be merged to master to make sure that RTD is still ok.

Reply:
Why isn't RTD building on this PR?

Reply:
Does not build on branch PRs.

Reply:
I don't seem to have access to drms on RTD so I can't fix this.